presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -100,9 +100,11 @@ Name                                   Description
 ``SSLKeyStorePath``                    The location of the Java KeyStore file that contains the certificate
                                        and private key to use for authentication.
 ``SSLKeyStorePassword``                The password for the KeyStore.
+``SSLKeyStoreType``                    The type of the KeyStore.
```

Comment:
We can put the default key store name `JKS` or just saying ["The default key store is provided by the Java security property files"](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/security/KeyStore.html#getDefaultType()).